worlds/factorio/__init__.py

```diff
@@ -43,89 +44,74 @@ class Factorio(World):
     research new technologies, and become more efficient in your quest to build a rocket and return home.
     """
     game: str = "Factorio"
-    static_nodes = {"automation", "logistics", "rocket-silo"}
+    special_nodes = {"automation", "logistics", "rocket-silo"}
     custom_recipes: typing.Dict[str, Recipe]
+    location_pool: typing.List[FactorioScienceLocation]
     advancement_technologies: typing.Set[str]
 
     web = FactorioWeb()
 
     item_name_to_id = all_items
-    location_name_to_id = base_tech_table
+    location_name_to_id = location_table
```

Comment:
this will break hinting for old seeds. we could leave the old info in, for 1 version at least, and put the new locations on new numbers. Kinda bad with the async running currently :S

Reply:
location hints were pretty much unusable anyway, as they're not part of the name. How much ID space do I have anyway till Factorio collides with the next thing?